### ISSUE TITLE
Reset timer if a taunt is started by the player

### ISSUE THREAD

**Issue by fladna9** [fladna9/Gmod-autotaunt#1]
Currently, when the player taunts, the timer is not reset. Should implement that.

**Comment by atikanc** [comment]
Good idea! Here is another addon designed for a reworked version of Prop Hunt, maybe use this for reference on how to do it?
https://github.com/mikesmullin/gmod-prop-hunters-auto-taunt
